ckanext/georchestra/controller.py

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+import six
+
+import ckan.lib.base as base
+import ckan.lib.helpers as h
+import ckan.plugins.toolkit as t
```

Comment:
Can you import in a pythonic way (`from ckan.libs import helpers`) make more sense than `h` which look like a local variable.

Reply:
ok